build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -412,6 +412,10 @@ message Action {
   // If true, then the `Action`'s result cannot be cached, and in-flight
   // requests for the same `Action` may not be merged.
   bool do_not_cache = 7;
+
+  // List of required supported [NodeProperty][build.bazel.remote.execution.v2.NodeProperty]
+  // keys. If a property is not recognized the server returns INVALID_ARGUMENT.
+  repeated string output_properties = 8;
```

Comment:
Is there a sorting requirement here? I don't necessarily think we need one, but we should be explicit about that if so.

Reply:
Oh, good catch, these absolutely have to be sorted. Note this is part of the Action, so order affects the cache key.